### ISSUE TITLE
`stubtest` contribution guide comes out non successful

### ISSUE THREAD

**Issue by flaeppe** [typeddjango/django-stubs#1627]
# Bug report

I tried to follow the [stubtest contribution guide](https://github.com/typeddjango/django-stubs/blob/master/CONTRIBUTING.md#testing-stubs-with-stubtest) but I can't manage to get a successful run. Not sure how to hunt down the problem either..

## What's wrong

Running the stubtest script via the suggested command `bash ./scripts/stubtest.sh` yields

```
➤ bash ./scripts/stubtest.sh
error: not checking stubs due to mypy build errors:
django-stubs/db/models/manager.pyi:10: error: Module "django_stubs_ext" has no attribute "ValuesQuerySet"
```

## How is that should be

There should be no error, [CI passed](https://github.com/typeddjango/django-stubs/actions/runs/5531541102/jobs/10092370164) on 2b1a4cdf8692bc2e88431d08fbe064665bb1b207

## System information

`python` version: 3.11.4

**Comment by sobolevn** [comment]
`pip install -r requirements.txt`?

**Comment by flaeppe** [comment]
I've tried that a couple of times, also recreated env. Also tried non-editable installs of `django-stubs` and `django-stubs-ext`, which resulted in other strange errors.

Something is weird, not sure what though..

**Comment by sobolevn** [comment]
Please, share repro steps :)

**Comment by flaeppe** [comment]
No idea really..

I made a clean clone to try to reproduce, only left me with missing `gis`/`GDAL` library errors..

I wonder what I've managed to break in my sibling directory 🙂

**Comment by UnknownPlatypus** [comment]
Had the same issue, I don't remember how I fixed it tho. I think it was clearing mypy cache.

**Comment by abdulaev-sh-m** [comment]
Here is a fix that should be made to prevent this problem from occurring again (I will add PR now): 
In the test_aliases.py file, change the import to this `from django_stubs_ext.django_stubs_ext import ValuesQuerySet`.

But this is not the main problem with the tests, since the result of running the `pytest` command is like this: 
`74 failed, 306 passed in 575.26s (0:09:35)`.
I guess I'll be fixing the tests in the near future)

P.S. I ran the tests on Huawei MateBoot D 15 (Windows 11 Home), Python version 3.11.4.

**Comment by flaeppe** [comment]
Closed via #1667
